src/commands/setup.rs

```diff
@@ -20,6 +20,11 @@ impl Setup {
     }
 
     pub fn exec(&self, input_file: String) {
+        assert!(
+            network::validation::ns_checks(&input_file),
```

Comment:
This has to validate network namespace so field should point to network namespace.
```suggestion
            network::validation::ns_checks(&self.network_namespace_path),
```